AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_AutoComponent_RPC.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+# Test Case Title : Check that the four network RPCs can be sent and received
+
+
+# fmt: off
+class Tests():
```

Comment:
Minor: Tests is not a very useful class name as this defines TestConstants at best.